tests/README.md

```diff
@@ -0,0 +1,37 @@
+# Helm template tests
+
+To add test for new template file, please create new directory with given structure:
+
+```text
+example_test/  # Test set name
+├── config.sh  # Configuration file
+└── static  # Test cases
+    ├── test_name.input.yaml  # Input configuration for test_name
+    └── test_name.output.yaml  # Output configuration for test_name
+```
+
+## Configuration file
+
+`config.sh` should contain `TEST_TEMPLATE` env variable, which should point to the helm template
+name, e.g. for `deploy/helm/sumologic/templates/configmap.yaml` it will be `templates/configmap.yaml`:
```

Comment:
```suggestion
`config.sh` should export `TEST_TEMPLATE` env variable, which should point to the helm template
file name, e.g. for `deploy/helm/sumologic/templates/configmap.yaml` it will be `templates/configmap.yaml`:
```